apps/hash-ai-worker-ts/docker/Dockerfile

```diff
@@ -1,40 +1,57 @@
-FROM node:22.12-alpine AS base
+FROM bitnami/minideb:bookworm AS mise
```

Comment:
nonblocking but I'd be cautious in choosing bitnami images - they're known for simply unannounced discontinuing and then removing images.

Reply:
Can you recommend an alternative?

Reply:
Should we just use Debian-slim?

Reply:
Yea I think so, otherwise it's always worth to take a look at chainguard as well. They also develop the apko and melange tools, but that's way out of scope.